scss/_variables.scss

```diff
@@ -787,6 +794,8 @@ $breadcrumb-padding-y:          .75rem !default;
 $breadcrumb-padding-x:          1rem !default;
 $breadcrumb-item-padding:       .5rem !default;
 
+$breadcrumb-margin:             0 0 1rem !default;
```

Comment:
@mdo Should I add breadcumb-margin too? I mean this PR is for paragraph-margin. For breadcrumb there's a different PR #23138 .